prov/verbs/src/windows/verbs_nd_ibv.c

```diff
@@ -363,29 +362,9 @@ int ibv_destroy_cq(struct ibv_cq *cq)
 	}
 
 	cq_nd = container_of(cq, struct nd_cq, cq);
-	ofi_mutex_lock(&cq_nd->lock);
-	if (cq_nd->notify_pending) {
-		hr = cq_nd->nd2cq->lpVtbl->CancelOverlappedRequests(
-			cq_nd->nd2cq);
-		if (FAILED(hr)) {
-			errno = hresult2fi(hr);
-			return errno;
-		}
-		ofi_mutex_unlock(&cq_nd->lock);
-		pthread_yield();
-
-		ofi_mutex_lock(&cq_nd->lock);
-		while (true) {
-			if (cq_nd->notify_pending) {
-				ofi_mutex_unlock(&cq_nd->lock);
-				usleep(1 << delay++);
-				ofi_mutex_lock(&cq_nd->lock);
-			} else {
-				break;
-			}
-		}
-	}
-	ofi_mutex_unlock(&cq_nd->lock);
+    if (cancel_pending(&cq_nd->notification, (IND2Overlapped*)cq_nd->nd2cq)) {
+        return errno;
+    }
```

Comment:
indentation, code uses tabs, with a tab being 8 spaces (but as a tab character, not spaces).